os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -440,9 +440,14 @@ static void _up_assert(int errorcode)
 static void recovery_user_assert(void)
 {
 	int ret;
+	mqd_t mqfd;
 	binmgr_request_t request_msg;
 	struct tcb_s *tcb;
 
+	/* Get mqfd for sending recovery mesage to binary manager */
+	mqfd = binary_manager_get_mqfd();
+	DEBUGASSERT(mqfd);
```

Comment:
I think DEBUGASSERT is not enough, because it is only enabled when debug config is enabled.

Reply:
right, I will modify